klippy/kinematics/cartesian.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, toolhead, config):
                                               above=0., maxval=max_velocity)
         self.max_z_accel = config.getfloat('max_z_accel', max_accel,
                                            above=0., maxval=max_accel)
+        self.ignore_homing = config.getsection('force_move').getboolean(
```

Comment:
what happens here when there is no section `force_move` in the config? I think the way this is implemented currently, klipper will error out if no such section is present.

Reply:
I tried that, it assumes the False value from getboolean()

Reply:
I see, thanks for the reply!